tests/pos/i2570.scala

```diff
@@ -0,0 +1,29 @@
+object Test {
+
+  def repeat(s: String, i: Int, j: Int = 22) = s * i
```

Comment:
In dotc you get:

    1 |repeat(1, _)(_)
      |             ^
      |             unbound placeholder parameter; incorrect use of `_`

In scalac you get:

    scala> repeat(1, _)(_) 
    <console>:13: error: missing parameter type for expanded function ((x$1: <error>, x$2) => repeat(1, x$1)(x$2))
           repeat(1, _)(_)
                     ^
    <console>:13: error: missing parameter type for expanded function ((x$1: <error>, x$2: <error>) => repeat(1, x$1)(x$2))
           repeat(1, _)(_)

The fact that the types already contain <error> markers makes me suspect that scalac also issues a parsing error but somehow suppresses the error message.